dockerfiles/cli/codenvy.sh

```diff
@@ -250,15 +236,15 @@ check_docker() {
 
   # If DOCKER_HOST is not set, then it should bind mounted
   if [ -z "${DOCKER_HOST+x}" ]; then
-      if ! docker ps >> "${LOGS}" 2>&1; then
-        info "Welcome to Codenvy!"
-        info ""
-        info "We did not detect a valid DOCKER_HOST." 
-        info ""
-        info "Rerun the CLI:"
-        info "  docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock "
-        info "                      -v <local-path>:/codenvy "
-        info "                         codenvy/cli $@"    
+      if ! docker ps > /dev/null 2>&1; then
+        printf "${GREEN}INFO:${NC} Welcome to Codenvy!\n"
```

Comment:
The info() command requires use of LOGS which is not yet initialized.  I needed to move log initialization until after check mounts and I wanted to make it clear that this output was raw in the bootstrap phase.